### DISCUSSION TITLE
更新日志 - 22.03.5 - 20231222

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1201] [Announcements]
__本次更新只包含RK35xx系列__

# 设备
* RK35xx：
  1. 移除 Intel WIFI 驱动（估计只有 x86 平台才需要）
  2. Hlink H28K 支持 GPU （Jellyfin 官方已经开始支持 RK35xx 系列的硬解，其中 tonemapping 功能可能依赖 GPU）

**Comment by lmf-cumt** [comment]
你好，我使用的设备时R6s，是需要Intel WIFI 驱动的，因为更新固件会导致无法连接无线网，所以希望能在之后的固件里面保留相关驱动。

**Reply by jjm2473** [reply]
你的wifi网卡是什么型号？AX201这些吗？

**Reply by lmf-cumt** [reply]
我使用的网卡是ax200

**Reply by jjm2473** [reply]
OK

**Comment by breeze303** [comment]
啥时候能升级下内核到6.×

**Reply by rhinogithub** [reply]
6.x内核是不是对新主机支持较好尤其虚拟化方面？如果是那似乎应该尽早升级期待下。

**Reply by jjm2473** [reply]
openwrt社区的release都还没到6.x呢，还在5.15

**Reply by breeze303** [reply]
目前只有lede的闭源版本能到6.6，6.1的话就已经有适配了

**Reply by jjm2473** [reply]
lean的内核版本比较多，有6.x
